File: Array & String/Remove Duplicates from Sorted Array/RemoveDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int j = 0;

        for (int i = 0; i < nums.length; i++) {
            if (i < nums.length - 1 && nums[i] == nums[i + 1]) {
                continue;
            } else {
                nums[j++] = nums[i];
            }
        }
        return j;
    }
}

File: Array & String/Remove Duplicates from Sorted Array/RemoveDuplicates.py
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i, j = 1, 1

        while i < len(nums):
            if nums[i - 1] != nums[i]:
                nums[j] = nums[i]
                j += 1
            i += 1

        return j
